packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-c3d4e5f6-a7b8-9012-cdef-012345678902.json

```
@@ -0,0 +1,225 @@
+        "ecs_mapping": [
+            {
+                "key": "event.category",
+                "value": {
+                    "value": ["configuration"]
+                }
+            },
+            {
+                "key": "event.type",
+                "value": {
+                    "value": ["info"]
+                }
+            },
+            {
+                "key": "event.kind",
+                "value": {
+                    "value": "state"
+                }
+            },
+            {
+                "key": "event.module",
+                "value": {
+                    "value": "osquery"
+                }
+            },
+            {
+                "key": "event.dataset",
+                "value": {
+                    "value": "osquery.launchd"
+                }
+            },
+            {
+                "key": "host.os.type",
+                "value": {
+                    "value": "macos"
+                }
+            },
+            {
+                "key": "rule.description",
+                "value": {
+                    "field": "detection_reason"
+                }
+            },
+            {
+                "key": "rule.name",
+                "value": {
+                    "field": "detection_method"
+                }
+            },
+            {
+                "key": "rule.category",
+                "value": {
+                    "field": "launchd_type"
+                }
+            },
+            {
+                "key": "service.state",
+                "value": {
+                    "field": "status"
+                }
+            },
+            {
+                "key": "service.name",
+                "value": {
+                    "field": "name"
+                }
+            },
+            {
+                "key": "process.executable",
+                "value": {
+                    "field": "executable_path"
+                }
+            },
+            {
+                "key": "process.command_line",
+                "value": {
+                    "field": "program_arguments"
+                }
+            },
+            {
+                "key": "file.path",
+                "value": {
+                    "field": "plist_path"
+                }
+            },
```

This is unrelated to all the other file details. Move to `event.provider`?

```suggestion
            {
                "key": "event.provider",
                "value": {
                    "field": "plist_path"
                }
            },
```